sdk/cosmos/azure-cosmos-kafka-connect/pom.xml

```diff
@@ -13,7 +13,7 @@ Licensed under the MIT License.
 
```

Comment:
We'd require an update to `azure-cosmos` version too?

Reply:
Resolving as `azure-cosmos` is shaded - https://github.com/Azure/azure-sdk-for-java/blob/a80eb9556de5471d47c3ca211c2ee9fe19cf68bc/sdk/cosmos/azure-cosmos-kafka-connect/pom.xml#L352-L358.

Reply:
No, azure-cosmos is shaded - see https://github.com/Azure/azure-sdk-for-java/blob/a80eb9556de5471d47c3ca211c2ee9fe19cf68bc/sdk/cosmos/azure-cosmos-kafka-connect/pom.xml#L352-L358